app/controllers/api/v2/works_controller.rb

```diff
@@ -205,7 +205,7 @@ def story_parser_options(archivist, work_params)
       collection_names: work_params[:collection_names],
       title: work_params[:title],
       fandom: work_params[:fandoms],
-      warning: work_params[:warnings],
+      archive_warning: work_params[:archive_warnings],
```

Comment:
@ariana-paris Is this change enough to make a v3 necessary?

Reply:
No, the only user of the API is Open Doors and this is a minor change from the client point of view (though one I'll have to make, of course, good catch!)